source/blog/2020-03-24-rspec-rails-4-0-has-been-released.md

```diff
@@ -0,0 +1,190 @@
+---
+title: RSpec Rails 4.0 has been released!
+author: Benoit Tigeot, Jon Rowe and Phil Pirozhkov
+---
+
+RSpec Rails 4.0 has been released! Given our commitment to [semantic versioning](http://semver.org/), this is a major release and as such contains breaking changes. The major breaking change is that we've dropped support for Rails below 5, with only soft support for Rails 4.2.
+
+As usual, if you notice any newly introduced bugs, please let us
+know, and we'll release a fix for it ASAP.
+
+RSpec continues to be a community-driven project with contributors
+from all over the world. This release includes 258 commits and 56
+merged pull requests from 27 different contributors!
+
+Thank you to everyone who helped make this release happen!
+
+## Notable Changes
+
+Support for Rails 6, including support for testing ActionCable, ActiveMailbox.
+
+### Breaking changes
+
+RSpec Rails 4 only supports Rails 5 and 6. This is in line with our
+new versioning policy which keeps major versions of RSpec Rails in
+lockstep with supported versions of Rails.
+
+This means that we no longer support versions of Ruby below 2.2
+in line with the supported Ruby versions of Rails 5.
+
+We have restricted the version of Rails for this version to 4.2 to
+allows a "soft support" for Rails 4.2 during this transitionary
+period but we strongly urge you to upgrade Rails to use 4.0.0.
+
+### Removed version lock with RSpec gems
+
+The `rspec-rails` gem is no longer locked with the versioning of the other RSpec gems, we will now be releasing `rspec-rails` with each new major Rails release, keeping in lockstep with their supported major versions.
+
+### Improved JRuby support
+
+We have improved JRuby support and RSpec Rails 4 fully supports JRuby on Rails 5 and 6.
+
+### Support for Action Cable
+
+We now support [channel specs][1] in order to easily test your Action Cable channels.  A `channel spec` pulls in `ActionCable::Channel::TestCase`, including all of the behavior and assertions that it provides, in addition to RSpec's own behavior and expectations.
+
+### Support for Action Mailbox
+
+Rails 6 came with [Action Mailbox][2]. We can now test them using
+`:mailbox` flag and use [mailbox specs][4].
+
+For example with `process` method and `have_been_delivered` matcher:
+
+~~~ ruby
+RSpec.describe InboxMailbox, type: :mailbox do
+  it "marks email as bounced when number tag in subject is valid" do
```

Comment:
This test doesn't mark the email as bounced?

Reply:
Yep. I fixed this error in the documentation too. Thanks